.github/workflows/ci-pipeline.yml

```diff
@@ -0,0 +1,107 @@
+name: CI Pipeline GitHub
+
+on: [push, pull_request]
+
+jobs:
+  backend-test:
+    runs-on: ubuntu-latest
+    env:
+      WORKDIR: backend
+      JWT_SECRET: ${{ github.sha }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+      - name: Install dependencies
+        run: cd $WORKDIR && npm install
```

Comment:
Have you tried to use [jobs.<job_id>.defaults.run.working-directory](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iddefaultsrunworking-directory) ?